.gitignore

```diff
@@ -1,2 +1,5 @@
 /build
 /node_modules
+.idea
```

Comment:
Please don't add IDE-specific ignore files. They can go in your `.git/info/exclude`.